templates/ctl-scaffold/test/Main.purs

```diff
@@ -52,4 +52,5 @@ config =
       , dbname: "ctxlib"
       }
   , customLogger: Nothing
+  , suppressLogs: false
```

Comment:
Please update the CTL revision in the template's flake and `packages.dhall`

Reply:
We can't update the revision until we know it (that is after we merge the PR)

Reply:
Not necessarily, you can set it to head on the branch you're working on